bundles/org.openhab.binding.publictransportswitzerland/README.md

```diff
@@ -0,0 +1,32 @@
+# Public Transport Switzerland Binding
+
+Connects to the "Swiss public transport API" to provide real-time public transport information. [Link to the API](https://transport.opendata.ch/)
+
+For example, here is a station board in HABPanel. (Download [here](https://github.com/StefanieJaeger/HABPanel-departure-board))
+
+![Departure board in HABPanel](doc/departure_board_habpanel.png)
+
+## Supported Things
+
+### Stationboard
+
+Upcoming departures for a single station.
```

Comment:
Supported Things and their config can be merged to a single chapter, since they are short.

Can you mention the Thing Type ID and the name, type and required/optional state of the config parameter? As the `station` value is a text, can you give the user a hint where to get the exact station name from? I can imagine there might be several spelling alternatives for a station name.

Reply:
Can you comment on this?